.pylintrc

```diff
@@ -66,9 +66,11 @@ disable=missing-docstring,
     unused-variable,
     pointless-string-statement,
     too-many-arguments,
+    too-many-positional-arguments,
     too-few-public-methods,
     useless-object-inheritance,
-    invalid-name
+    invalid-name,
+    cyclic-import
```

Comment:
Might be a better idea to only disable this linting on the line that cause the cyclic import, I guess it is the flask route import which is only in this special case ok ?

Reply:
👍🏼